### ISSUE TITLE
Test error: ErrWriteAfterFlush is no longer returned by anything in the net/http package. Callers should not compare errors against this variable.

### ISSUE THREAD

**Issue by eclipseo** [dominikh/go-tools#350]
```
Testing: "/builddir/build/BUILD/go-tools-2017.2.2/_build/src/honnef.co/go/tools/staticcheck"
+ GOPATH=/builddir/build/BUILD/go-tools-2017.2.2/_build:/usr/share/gocode
+ go test -buildmode pie -compiler gc -ldflags '-extldflags '\''-Wl,-z,relro -Wl,--as-needed  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld '\'''
--- FAIL: TestAll (35.97s)
    util.go:118: Lint failed at CheckDeprecated_go14.go:18; /ErrWriteAfterFlush is no longer used/ did not match
    util.go:126: Unexpected problem at testdata/CheckDeprecated_go14.go:18:12: http.ErrWriteAfterFlush is deprecated: ErrWriteAfterFlush is no longer returned by anything in the net/http package. Callers should not compare errors against this variable. 
    util.go:118: Lint failed at CheckDeprecated_go18.go:18; /ErrWriteAfterFlush is no longer used/ did not match
    util.go:126: Unexpected problem at testdata/CheckDeprecated_go18.go:18:12: http.ErrWriteAfterFlush is deprecated: ErrWriteAfterFlush is no longer returned by anything in the net/http package. Callers should not compare errors against this variable. 
FAIL
exit status 1
FAIL	honnef.co/go/tools/staticcheck	36.219s
```

I'm using this revision of net/http https://github.com/golang/net/tree/db08ff08e8622530d9ed3a0e8ac279f6d4c02196 with Golang 1.11. And version 2017.2.2 of go-tools.

**Comment by dominikh** [comment]
This will be fixed once we merge the `next` branch (which will be once an upstream issue in go/packages is fixed) and release 2018.1

**Comment by eclipseo** [comment]
Fixes in 2019.1. Thanks.
